### PR TITLE
Check that enumerateDevices is returning new objects for every call

### PR DESCRIPTION
**Pull request by youennf**

### CODE REVIEW

**Comment by foolip** [comment]
https://wpt.fyi/results/mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html?sha=ac25c4fb56&label=pr_head shows this passing in all browsers, is that expected?

File: mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            await navigator.mediaDevices.getUserMedia({audio : true, video: true});
+
+        const deviceList1 =  await navigator.mediaDevices.enumerateDevices();
+        const deviceList2 =  await navigator.mediaDevices.enumerateDevices();
+
+        assert_equals(deviceList1.length, deviceList2.length);
+        for (let i = 0; i < deviceList1.length; i++) {
+            const device1 = deviceList1[i];
+            const device2 = deviceList2[i];
+            assert_false(device1 === device2);
```
Comment (foolip) [inline mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html]:
```suggestion
            assert_not_equals(device1, device2);
```

**Review by foolip — APPROVED** [review]
Just one suggestion.

**Comment by youennf** [comment]
> https://wpt.fyi/results/mediacapture-streams/MediaDevices-enumerateDevices-returned-objects.https.html?sha=ac25c4fb56&label=pr_head shows this passing in all browsers, is that expected?

Yes, the plan is to  align the spec with implementations.

**Comment by foolip** [comment]
Alright, please merge whenever you like :)

**Pull request merged**